deadline_/notifications/router.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+import json
+import logging
+
+from .channels import user_authentication, read_notification
+
+logger = logging.getLogger('notifications')
+
+
+class MessageRouter(object):
```

Comment:
Why MessageRouter inherits from object? I thought we don't do that in python3.x.

Reply:
Yep, good catch. This is what you get from CDD (Copy-Paste Driven Development)